File: redis/erlang/delete/redis_delete.erl
#! /usr/bin/escript
%% -----------------------------------------------------------
%%	redis_delete.erl
%%
%%					May/20/2013
%% -----------------------------------------------------------
-module(socket).
%% -----------------------------------------------------------
main (A)->
	io:format ("*** 開始 ***\n"),
	[Key_in]=A,
	io:format (lists:concat (["Key_in = ",Key_in,"\n"])),
%%
	Host = "host_dbase",
	Port = 6379,
	{ok, Sock} = gen_tcp:connect(Host, Port,
                                 [binary, {packet, 0}]),
	Command =  lists:concat(["del ",Key_in, "\r\n"]),
	ok = gen_tcp:send(Sock, Command),
	ok = gen_tcp:close(Sock),
%%
	io:format ("*** 終了 ***\n").
%% -----------------------------------------------------------

File: redis/go/delete/redis_delete.go
// ----------------------------------------------------------------
//
//	redis_delete.go
//
//					May/19/2018
//
// ----------------------------------------------------------------
package main
 
import (
	"fmt"
	"net"
	"os"
)

// ----------------------------------------------------------------
func main () {

	fmt.Printf ("*** 開始 ***\n")

	key_in := os.Args[1]

	fmt.Printf (key_in + "\n")

	hostname := "localhost"
	port := "6379"

	conn, err := net.Dial ("tcp",hostname + ":" + port)
	if err != nil {
		fmt.Println(err)
		return
		}

	_, err = conn.Write ([]byte("del " + key_in + "\r\n"))
	if err != nil {
		fmt.Println(err)
		return
		}

	conn.Close ()

	fmt.Printf ("*** 終了 ***\n")
}

// ----------------------------------------------------------------
